config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -1682,6 +1683,16 @@ public AuthorizeExchangeSpec authenticated() {
 				return access(AuthenticatedReactiveAuthorizationManager.authenticated());
 			}
 
+			/**
+			 * Require a specific IP Address or subnet
```

Comment:
Perhaps add a bit more detail, similar to the original `WebSecurityExpressionRoot`. Something like `Require a specific IP address or range using an IP/Netmask (e.g. 192.168.1.0/24)`.

Reply:
Done.